l5kit/l5kit/environment/callbacks.py

```diff
@@ -83,6 +83,17 @@ def _on_step(self) -> bool:
                 self.logger.record(f'{self.prefix}/ade', ade_error)
                 self.logger.record(f'{self.prefix}/fde', fde_error)
 
+            # Calculate replayed/simulated driven miles
+            if 'replay_driven_miles' in self.metric_set.evaluation_plan.metrics_dict():
```

Comment:
I think we should check if we have these keys during the construction of this callback and trigger an error if the metric set doesn't have the required metrics/cms/etc.